airflow/dag_processing/manager.py

```diff
@@ -393,7 +393,7 @@ def __init__(
         self._pickle_dags = pickle_dags
         self._dag_ids = dag_ids
         self._async_mode = async_mode
-        self._parsing_start_time: int | None = None
+        self._parsing_start_time: float | None = None
```

Comment:
this is a freebie while I'm at it. We fill this field here, with a fractional number of seconds. 
https://github.com/apache/airflow/blob/9979cb62ac1631f5fd372c17b531c4df1a87e293/airflow/dag_processing/manager.py#L1117